arm-datalake-analytics/job/2016-11-01/swagger/job.json

```diff
@@ -700,7 +700,10 @@
           "enum": [
             "Warning",
             "Error",
-            "Info"
+            "Info",
```

Comment:
@amarzavery @markcowl I don't think this is truly a breaking change candidate since the new members being added to swagger have always been valid in the service (which was found because of a client break). This is a bug fix for missing enum values vs. a feature addition of new values in both the service and client.

Basically, just the client was missing these values as possibilities to be returned from the service. This is similar to other fixes we have made to make our swagger spec more accurately reflect the reality of the service as we move toward convergence between the client and service contract sharing the swagger spec (which has not yet happened, but we are getting closer every day)